src/scheduler/fifo.c

```diff
@@ -479,14 +479,14 @@ init_scheduling_cycle(status *policy, int pbs_sd, server_info *sinfo)
  *		   It will handle the difference cases that caused a
  *		   scheduling cycle
  *
- * @param[in]	sconn	-	connection and command info from server
+ * @param[in]	sd	-	primary socket descriptor to the server pool
  *
  * @return	int
  * @retval	0	: continue calling scheduling cycles
  * @retval	1	: exit scheduler
  */
 int
-schedule(sched_svrconn *sconn, sched_cmd *cmd)
+schedule(int sd, sched_cmd *cmd)
```

Comment:
Do we really need sd as argument? now even with multi-server we have only one sd only 'cluster fd', and which is global variable anyway. So can we get rid of sd argument (including all dependent funcs) and use global cluster fd in IFL calls during sched cycle?

Reply:
@arungrover We can set global cluster fd variable to simulation fd.

Reply:
Ah, got it.